pkg/R/sparkR.R

```diff
@@ -15,7 +15,7 @@ sparkR.onLoad <- function(libname, pkgname) {
 # backend and FALSE otherwise
 connExists <- function(env) {
   tryCatch({
-    exists(".sparkRCon", envir = env) && isOpen(env[[".sparkRCon"]])
+    !exists(".sparkRCon", envir = env) || (exists(".sparkRCon", envir = env) && isOpen(env[[".sparkRCon"]]))
```

Comment:
@sun-rui  This was necessary because if sparkR.stop() is inadvertently called multiple times or called before exiting R an error occurs, which I paste below. The error stems from the fact that sparkR.stop() removes ".sparkRCon"

>sc <- sparkR.init("local")
>sparkR.stop()
>sparkR.stop()
Error in socketConnection(host = hostname, port = port, server = FALSE,  : 
  cannot open the connection 
3 socketConnection(host = hostname, port = port, server = FALSE, 
    blocking = TRUE, open = "wb", timeout = timeout) at sparkRClient.R#11
2 connectBackend("localhost", .sparkREnv$sparkRBackendPort) at sparkR.R#34
1 sparkR.stop() 
In addition: Warning message:
In socketConnection(host = hostname, port = port, server = FALSE,  :
  localhost:12345 cannot be opened